pandas_vet/__init__.py

```diff
@@ -228,6 +232,28 @@ def check_for_read_table(node: ast.Call) -> List:
     return []
 
 
+def check_for_merge(node: ast.Call) -> List:
+    """
+    Check for use of `.merge()` method on the pandas object.
+
+    Error/warning message to recommend use of `df.merge()` method instead.
+    """
+    # The AST does not retain any of the pandas semantic information, so the
+    # current implementation of this test will infer based on the name of the
+    # object.  If the object name is `pd`, and if the `.merge()` method has at
+    # least two arguments (left, right, ... ) we will assume that it matches 
+    # the pattern that we are trying to check, `pd.merge(left, right)`
+    #if not isinstance( XXX node parent XXX, 'module'): return []
+    if not hasattr(node.func, 'value'): return []   # ignore functions
+    if not node.func.value.id == 'pd': return[]     # assume object name is `pd`
+    if not len(node.args) >= 1: return []           # at least two arguments
```

Comment:
Should be >= 2, no?

```suggestion
    if not len(node.args) >= 2: return []           # at least two arguments
```

Reply:
Yup, good catch.  I originally had this nested in the below if block, but moved it above with a `not` to make it more readable, and missed the change.